cpp/installer.cpp

```diff
@@ -233,19 +233,24 @@ void install(jsi::Runtime &rt, std::shared_ptr<react::CallInvoker> jsCallInvoker
       {
         if (sizeof(args) < 3)
         {
-          return createError(rt, "[react-native-quick-sqlite][execSQLBatch] - Incorrect parameter count");
+          return {};
```

Comment:
please throw an error, no better way to make sure users don't notice something being wrong